cmd/manager/main.go

```diff
@@ -204,3 +218,30 @@ func serveCRMetrics(cfg *rest.Config) error {
 	return nil
 }
 */
+func IsTopologyInUse(ctx context.Context) (bool, error) {
+	kubeConfig, err := rest.InClusterConfig()
```

Comment:
all these "client init" lines seem like a duplication of those in [getServerVersion](https://github.com/IBM/ibm-block-csi-operator/blob/7f0f44b70ffb6d72d98c6fcb5fa3734756f0e08f/pkg/controller/ibmblockcsi/ibmblockcsi_controller.go#L76) - could you try to unify them, and make them run only once?

Reply:
done